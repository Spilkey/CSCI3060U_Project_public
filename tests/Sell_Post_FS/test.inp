loginuser02sellairpods meetup100.0026buyairpods meetup4user02logout