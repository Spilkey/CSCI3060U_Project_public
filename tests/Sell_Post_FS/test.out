03 airpods meetup            user02        026 100.00
04 airpods meetup            user02        010 100.00
00 user02          FS 000000.00